src/components/ReportActionItem/MoneyReportView.js

```diff
@@ -59,10 +67,50 @@ function MoneyReportView(props) {
                         numberOfLines={1}
                         style={[styles.taskTitleMenuItem, styles.alignSelfCenter]}
                     >
-                        {formattedAmount}
+                        {formattedTotalAmount}
                     </Text>
                 </View>
             </View>
+            {shouldShowBreakdown ? (
+                <>
+                    <View style={[styles.flexRow, styles.pointerEventsNone, styles.containerWithSpaceBetween, styles.ph5, styles.pv1]}>
+                        <View style={[styles.flex1, styles.justifyContentCenter]}>
+                            <Text
+                                style={[styles.textLabelSupporting]}
+                                numberOfLines={1}
+                            >
+                                {translate('cardTransactions.outOfPocket')}
+                            </Text>
+                        </View>
+                        <View style={[styles.flexRow, styles.justifyContentCenter]}>
+                            <Text
+                                numberOfLines={1}
+                                style={subAmountTextStyles}
+                            >
+                                {formattedOutOfPocketAmount}
+                            </Text>
+                        </View>
+                    </View>
+                    <View style={[styles.flexRow, styles.pointerEventsNone, styles.containerWithSpaceBetween, styles.ph5, styles.pv1]}>
+                        <View style={[styles.flex1, styles.justifyContentCenter]}>
+                            <Text
+                                style={[styles.textLabelSupporting, StyleUtils.getColorStyle(themeColors.textSupporting)]}
```

Comment:
```suggestion
                                style={[styles.textLabelSupporting]}
```

NAB. `getColorStyle` is redundant. The color style is already a part of `textLabelSupporting`